stubs/python-dateutil/dateutil/relativedelta.pyi

```diff
@@ -87,11 +88,11 @@ class relativedelta:
     def __neg__(self) -> Self: ...
     def __bool__(self) -> bool: ...
     def __nonzero__(self) -> bool: ...
-    def __mul__(self, other: SupportsFloat) -> Self: ...
-    def __rmul__(self, other: SupportsFloat) -> Self: ...
+    def __mul__(self, other: ConvertibleToFloat) -> Self: ...
+    def __rmul__(self, other: ConvertibleToFloat) -> Self: ...
     def __eq__(self, other: object) -> bool: ...
     def __ne__(self, other: object) -> bool: ...
-    def __div__(self, other: SupportsFloat) -> Self: ...
-    def __truediv__(self, other: SupportsFloat) -> Self: ...
+    def __div__(self, other: ConvertibleToFloat) -> Self: ...
+    def __truediv__(self, other: ConvertibleToFloat) -> Self: ...
```

Comment:
While technically it would _work_, I don't think there would ever be a good reason to try to multiply or divide a `relativedelta` object by a string `"10"` (for example), would there? I think the current annotation is better here.